book/src/coding-conventions.md

```diff
@@ -70,3 +70,16 @@ TODO: how to name error types/error enum variants (`CannotDoSth` vs `DoingSth`)?
 ## Dependencies
 
 If the same dependency is used in multiples crates within the workspace, that dependency SHOULD be specified in the *workspace*'s `Cargo.toml` file and workspace crates should import them from there.
+
+## Adding a new workspace crate, exposed by `riot-rs`
+
+To add a new workspace crate re-exported by `riot-rs`, follow these steps:
+
+1. Create the new crate's directory in `src/`.
+1. Run `cargo init --lib` in that directory.
+1. Add relevant lints (e.g., `#![deny(missing_docs)]` and `#![deny(clippy::pedantic)]`) to the crate; some lints are already inherited from the workspace and do not need to be added to the new crate.
```

Comment:
I think this needs to be more specific. Better state e.g., "Add `#![deny(missing_docs)]` and `#![deny(clippy::pedantic)]` to the crate"